spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/cassandra/CassandraDataAutoConfiguration.java

```diff
@@ -80,10 +113,41 @@ public CassandraSessionFactoryBean session(CassandraConverter converter)
 		session.setCluster(this.cluster);
 		session.setConverter(converter);
 		session.setKeyspaceName(this.properties.getKeyspaceName());
-		session.setSchemaAction(SchemaAction.NONE);
+		session.setSchemaAction(SchemaAction.valueOf(this.properties.getSchemaAction()));
```

Comment:
Exactly and if you follow the discussion it ends up in #4890 where the idea is to indeed store the String value but have a service to apply the same binding programmatically.